crates/router/src/routes/app.rs

```diff
@@ -243,7 +243,11 @@ impl Refunds {
 
         #[cfg(feature = "olap")]
         {
-            route = route.service(web::resource("/list").route(web::get().to(refunds_list)));
+            route = route.service(
+                web::resource("/list")
+                    .route(web::get().to(refunds_list))
+                    .route(web::post().to(refunds_list_with_filters)),
```

Comment:
What's the need for a POST endpoint? Can't we use a GET endpoint (maybe even the existing one?) for this purpose?